src/com/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -288,9 +289,10 @@
     ;; confused if the database doesn't exist but we open and close a
     ;; connection without creating anything.
     (sql/with-connection write-db
-                         (scf-store/validate-database-version #(System/exit 1))
+                         (scf-store/validate-database-version
```

Comment:
Should we even do this validation for `enterprise?` since we can be sure they're using a version of Postgres which is compatible?